pom.xml

```diff
@@ -256,9 +256,9 @@
 					<execution>
 						<id>build-qualifier</id>
 						<phase>validate</phase>
-						<goals>
+						<!--<goals>
```

Comment:
Hi @thegreystone, This is the main change which is making build success. I am not sure on the impact. This generates the build qualifier according to the [rules described in the PDE documentation](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.pde.doc.user%2Ftasks%2Fpde_version_qualifiers.htm). 
I just compared the build with and without this goal. There is no difference in the target.

Reply:
Did you try to make a release build with these changes? Could you try building a release build and check what the about box says?

Azul Mission Control 9.0.0 CA
(Build id: 15, Qualifier: 202405211355, Change id: f58490a6bddde45d8862aab25795ebdb3b1b4d08)

Reply:
Sure. I will check this.